### PR TITLE
feat(http): add swagger documentation to platform

### PR DESCRIPTION
**Pull request by desa**
This PR adds swagger documentation for the dashboard API.

### CODE REVIEW

File: http/swagger.yml [hunk 1]
```diff
@@ -1,10 +1,226 @@
...
+          name: organization
+          schema:
+            type: string
+          description: filter for dashboards belonging to a particular organization
+        - in: query
+          name: organizationID
+          schema:
+            type: string
+          description: filter for dashboards belonging to a particular organizationID
+      responses:
```
Comment (goller) [inline http/swagger.yml]: Need to add a default response for errors

File: http/swagger.yml [hunk 2]
```diff
@@ -1,10 +1,226 @@
...
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  '/dashboards/{dashboardID}/cells':
+   post:
+      tags:
+        - Dashboards
+        - Cells
+      summary: all a dashboards cells
```
Comment (goller) [inline http/swagger.yml]: all dashboard cells

File: http/swagger.yml [hunk 3]
```diff
@@ -1,10 +1,226 @@
...
+              schema:
+                $ref: "#/components/schemas/Error"
+  '/dashboards/{dashboardID}/cells/{cellID}':
+   put:
+      tags:
+        - Dashboards
+        - Cells
+      summary: a dashboard cell
+      requestBody:
+          description: dashboard cell to create
```
Comment (goller) [inline http/swagger.yml]: dashboard cell to replace

File: http/swagger.yml [hunk 4]
```diff
@@ -591,6 +807,56 @@ paths:
...
+          format: int32
+        visualization:
+          oneOf:
+            - $ref: "#/components/schemas/CommonVisualization"
+    Dashboard:
+      properties:
+        id:
+          readOnly: true
+          type: string
+        organizationID:
```
Comment (goller) [inline http/swagger.yml]: Perhaps this would be a link

File: http/swagger.yml [hunk 5]
```diff
@@ -1,10 +1,226 @@
...
+      summary: Get a dashboard by id
+      parameters:
+        - in: path
+          name: dashboardID
+          schema:
+            type: string
+          required: true
+          description: ID of dashboard to get
+      responses:
+        '200':
```
Comment (jademcgough) [inline http/swagger.yml]: missing default response here as well

File: http/swagger.yml [hunk 6]
```diff
@@ -591,6 +807,56 @@ paths:
...
+          type: integer
+          format: int32
+        w:
+          type: integer
+          format: int32
+        h:
+          type: integer
+          format: int32
+        visualization:
+          oneOf:
```
Comment (jademcgough) [inline http/swagger.yml]: can probably drop the oneOf until there's actually more than one component schema to choose from

Comment (desa) [reply http/swagger.yml]: I think this might be pretty soon. Do you think I should still drop this? Happy to defer to your judgement

Comment (jademcgough) [reply http/swagger.yml]: nah you can leave it in

**Comment by desa** [comment]
closing this in favor of https://github.com/influxdata/platform/pull/126/files

**Pull request closed without merging**